App/Containers/Example/ExampleScreen.js

```diff
@@ -61,6 +63,7 @@ class ExampleScreen extends React.Component {
               onPress={() => this._fetchUser()}
               title="Refresh"
             />
+            <ProgressBar progress={0.34}/>
```

Comment:
was just more for seeing what the progress bar looks like? i'm not sure if we should merge this into master if so